Verification/SPIDVerificationAPI_WPF_Sample/EnrollPage.xaml.cs

```diff
@@ -54,6 +54,7 @@ public partial class EnrollPage : Page
         private string _subscriptionKey;
         private Guid _speakerId = Guid.Empty;
         private int _remainingEnrollments;
+        private const int maxRemainingEnrollments = 3;
```

Comment:
This is still hard-coded. The point is that the number of required enrollments cannot be hard-coded in the sample application because if this changes on the server side, the sample application will not work.